src/pyglottolog/languoids/models.py

```diff
@@ -186,7 +186,31 @@ def _format(self, minimal=True):
 
     @classmethod
     def from_name(cls, name):
+        # some additional common names
+        _code_map = {
+            'burma': 'MM', 'cameroun': 'CM', 'czech republic': 'CZ',
```

Comment:
Can you maybe provide a pointer where exactly these are not found (IIRC we only use two-letter codes in the glottolog repository)?

Reply:
It's so far only used while updating ElCat information [https://github.com/glottolog/pyglottolog/pull/61]. During the update it adds countries mentioned in ElCat if Glottolog has **no** countries at all.

Reply:
Ah, in that case, I'd vote to not include this. That's too many special handling piled on top of each other.